src/translations.js

```diff
@@ -268,6 +277,7 @@ const wakatimeCardLocales = {
     sk: "Wakatime štatistika",
     tr: "Waketime İstatistikler",
     pl: "statystyki Wakatime",
+    uz: "Vaqt holati",
```

Comment:
Here Google translate suggests `Vakatime statistikasi`.

Reply:
`Wakatime statistikasi` 
I think that's right now :grin: